### PR TITLE
Update password restriction matching to db image manual

### PR DESCRIPTION
**Pull request by jasonyang-ee**
per linuxserver/mariadb/, the required DATABASE_PASSWORD must > 4

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [O ] I have read the [contributing](https://github.com/linuxserver/docker-bookstack/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

The user manual on this repo did not sync with it's dependency https://hub.docker.com/r/linuxserver/mariadb/.

Per mariadb image, the required DATABASE_PASSWORD must greater than 4 characters.

## Benefits of this PR and context:

New user who sets password less than 4 char will cause error similar to database connection issue where previous issues are all arguing DB_PASS or DB_PASSWORD which is not the solution here.

In short, it is extremely difficult to troubleshoot this error. Eventually, I decided to RTFM for one layer up which discovered the solution. This update to README should help preventing future user to pull their hair out.

** PR rule says don't PR for README typo. But this is manual content change. :)

## How Has This Been Tested?

Test: setting longer password worked and short password does not.
Hardware: Raspberry PI 4B.
OS: Raspberry PI OS lite, arm64/v8

## Source / References:

https://github.com/linuxserver/docker-mariadb#parameters

### CODE REVIEW

**Comment by jasonyang-ee** [comment]
hmm, yeah, I didn't read the guide. I will do it on the [readme-vars.yml](https://github.com/linuxserver/docker-bookstack/edit/master/readme-vars.yml).
